applications/plugins/image/imageConfig.cmake.in

```diff
@@ -3,29 +3,23 @@
 @PACKAGE_GUARD@
 @PACKAGE_INIT@
 
-set(IMAGE_HAVE_OPENGL @IMAGE_HAVE_OPENGL@)
-set(IMAGE_HAVE_SOFAGUIQT @IMAGE_HAVE_SOFAGUIQT@)
-set(IMAGE_HAVE_CIMGPLUGIN @IMAGE_HAVE_CIMGPLUGIN@)
+set(IMAGE_HAVE_SOFA_GL @IMAGE_HAVE_SOFA_GL@)
+set(IMAGE_HAVE_SOFA_GUI_QT @IMAGE_HAVE_SOFA_GUI_QT@)
 set(IMAGE_HAVE_SOFAPYTHON @IMAGE_HAVE_SOFAPYTHON@)
 set(IMAGE_HAVE_MULTITHREADING @IMAGE_HAVE_MULTITHREADING@)
 set(IMAGE_HAVE_ZLIB @IMAGE_HAVE_ZLIB@)
 set(IMAGE_HAVE_FREENECT @IMAGE_HAVE_FREENECT@)
-set(IMAGE_HAVE_DIFFUSIONSOLVER @IMAGE_HAVE_DIFFUSIONSOLVER@)
 
-find_package(Sofa.GL REQUIRED)
+find_package(Sofa.Core QUIET REQUIRED)
+find_package(Sofa.Component.Visual QUIET REQUIRED)
+find_package(CImgPlugin QUIET REQUIRED)
 
-find_package(SofaBase REQUIRED)
-find_package(SofaGeneralVisual REQUIRED)
-find_package(Sofa.GL REQUIRED)
 
-if(IMAGE_HAVE_OPENGL)
-    find_package(OpenGL QUIET REQUIRED)
+if(IMAGE_HAVE_SOFA_GL)
+    find_package(Sofa.GL QUIET REQUIRED)
 endif()
 if(IMAGE_HAVE_SOFAGUIQT)
-    find_package(SofaGuiQt QUIET REQUIRED)
-endif()
-if(IMAGE_HAVE_CIMGPLUGIN)
-    find_package(CImgPlugin QUIET REQUIRED)
```

Comment:
Nope, in the main Cmake file, CImgPlugin is a *mandatory* dependency so no need to test if image found it, it will always find it (or fail at the configure time).